src/terrain/index.js

```diff
@@ -0,0 +1,157 @@
+const { getRandomTerrainFunction } = require('./terrainFunctions');
+const terrainGenerator = (boardBounds = { x: 100, y: 100 }, terrainBounds = { x: 5, y: 5 }, numberOfTerrainElements = 5) => {
+    let { mockArray, mockWidth, mockHeight } = createMockArray(boardBounds, terrainBounds);
+    const keyPoint = {
+        x: Math.floor(Math.random() * mockWidth),
+        y: Math.floor(Math.random() * mockHeight)
+    };
+    mockArray = generateTerrain(mockArray, keyPoint, numberOfTerrainElements, mockWidth, mockHeight);
+    return mockArray;
+};
+
+createMockArray = (boardBounds, terrainBounds) => {
```

Comment:
This isn't really a "mock" if I understand correctly, but more like an "initialized" array with some reasonable default values. So I guess I'd rename this to `initializeBoard` or something like that. 

Also, it's confusing to me why you would return width + height of the board you're returning when you can calculate that from both the args to the function as well as the result of it. I'd prefer it just returns the array.

Another also, should this be `const createMockArray = ...`?

Reply:
honestly I was just being lazy :P good call there

and nice catch with me missing `const`